pystac/extensions/pointcloud.py

```diff
@@ -0,0 +1,347 @@
+
+    def set_schemas(self, schemas, asset=None):
+        """Set an Item or an Asset schema
+
+        If an Asset is supplied, sets the property on the Asset.
+        Otherwise sets the Item's value.
+        """
+        if asset is None:
+            self.item.properties['pc:schemas'] = schemas
+        else:
+            asset.properties['pc:schemas'] = schemas
+
+    @property
+    def density(self):
+        """Get or sets the density for the item.
+
+        Density is defined as the number of points per square unit area.
+
+        Returns:
+            int
+        """
+        return self.get_density()
+
+    @density.setter
+    def density(self, v):
+        self.set_density(v)
+
+    def get_density(self, asset=None):
+        """Gets an Item or an Asset density.
+
+        If an Asset is supplied and the Item property exists on the Asset,
+        returns the Asset's value. Otherwise returns the Item's value
+
+        Returns:
+            int
+        """
+        if asset is None or 'pc:density' not in asset.properties:
+            return self.item.properties.get('pc:density')
+        else:
+            return asset.properties.get('pc:density')
+
+    def set_density(self, density, asset=None):
+        """Set an Item or an Asset density property.
+
+        If an Asset is supplied, sets the property on the Asset.
+        Otherwise sets the Item's value.
+        """
+        if asset is None:
+            self.item.properties['pc:density'] = density
+        else:
+            asset.properties['pc:density'] = density
+
+    @property
+    def statistics(self):
+        """Get or sets the statistics for each property of the dataset.
+
+        A sequential array of items mapping to pc:schemas defines per-channel statistics.
+
+        Exmample::
+
+            item.ext.pointcloud.statistics = [{ 'name': 'red', 'min': 0, 'max': 255 }]
+
+        Returns:
+            List[dict]
+        """
+        return self.get_statistics()
+
+    @statistics.setter
+    def statistics(self, v):
+        self.set_statistics(v)
+
+    def get_statistics(self, asset=None):
+        """Gets an Item or an Asset centroid.
+
+        If an Asset is supplied and the Item property exists on the Asset,
+        returns the Asset's value. Otherwise returns the Item's value
+
+        Returns:
+            dict
+        """
+        if asset is None or 'pc:statistics' not in asset.properties:
+            return self.item.properties.get('pc:statistics')
+        else:
+            return asset.properties.get('pc:statistics')
+
+    def set_statistics(self, statistics, asset=None):
+        """Set an Item or an Asset centroid.
+
+        If an Asset is supplied, sets the property on the Asset.
+        Otherwise sets the Item's value.
+        """
+        if asset is None:
+            self.item.properties['pc:statistics'] = statistics
+        else:
+            asset.properties['pc:statistics'] = statistics
+
+    @property
+    def epsg(self):
```

Comment:
EPSG is another extension - this may be leftover from using the `proj` extension as a template?

Reply:
yup this was my bad, proj extension handles this